definitions/npm/@reach/router_v1.1.x/flow_v0.104.x-/router_v1.1.x.js

```diff
@@ -71,7 +71,7 @@ declare module '@reach/router' {
   |}>;
 
   declare export type LinkProps<State> = {
-    ...$Shape<HTMLAnchorElement>,
+    ...React$ElementProps<string>,
```

Comment:
Why not `...React$ElementProps<a>`?

Reply:
`@reach/router` spreads `Link` props over `<a/>` element: https://github.com/reach/router/blob/master/src/index.js#L385-L398

Reply:
My thought was to make it more generic. I could imagine that `Link` component would render another element like `span`.

Reply:
`@reach/router` is very accessibility-centric, so I don't think they will ever allow using spans as links

Reply:
They actually even advertise using native links as a feature =)
https://reach.tech/router/accessibility